client/src/components/Auth/AuthenticationResponse.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { StyleSheet, Text, Dimensions, TouchableOpacity} from "react-native";
+import { FirebaseError } from "firebase/app";
+import { User } from "firebase/auth";
+
+//Type to handle Authentication Responses from firebase
+export type AuthenticationResponse = {
+  user: User | null;
+  error?: undefined;
+} | {
+  user?: undefined;
+  error: unknown;
+}
+
+class CustomError {
+  public code: string;
+  public message: string;
+
+  constructor(code: string, message: string) {
+    this.code = code;
+    this.message = message;
+  }
+}
+
+//Custom responses
+export const inValidEmailResponse = new CustomError("Invalid Email", "Please provide a valid email address")
+
+//Function that decodes the error code
+function decodeFirebaseError(error: FirebaseError) {
```

Comment:
Could you turn this `function` into an arrow function? (Refer to styling in docs)

Reply:
Resolved